### PR TITLE
Tofe/pinetab2

### PR DESCRIPTION
**Pull request by Tofee**

### CODE REVIEW

File: recipes-bsp/u-boot/u-boot_%.bbappend [hunk 1]
```diff
@@ -26,6 +26,8 @@ SRC_URI:append:pinephone = " \
     file://0001-pinephone-Add-volume_key-environment-variable.patch \
 "
 UBOOT_MAKE_TARGET:pinephone = "pinephone_defconfig all"
+UBOOT_MAKE_TARGET:pinetab2 = "pinetab2-rk3566_defconfig all"
```
Comment (Tofee) [inline recipes-bsp/u-boot/u-boot_%.bbappend:29]: I just realized this line isn't needed, as we already use UBOOT_MACHINE variable in the machine conf file.

Comment (Herrie82) [reply recipes-bsp/u-boot/u-boot_%.bbappend:29]: @tofee will you remove it here then?

Comment (Herrie82) [reply recipes-bsp/u-boot/u-boot_%.bbappend:29]: @Tofee LGTM for the rest, all works

Comment (Tofee) [reply recipes-bsp/u-boot/u-boot_%.bbappend:29]: Yes I'll amend this commit

Comment (Tofee) [reply recipes-bsp/u-boot/u-boot_%.bbappend]: Ok, done

**Pull request merged**